web/satellite/src/api/baseRest.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+import { AuthToken } from '@/utils/authToken';
+
+/**
+ * BaseRest is an utility which allows to perform web requests
+ */
+export class BaseRest {
```

Comment:
Rest - is a pattern name. not a transport or language.

Its better to create http utility